flagd/pkg/runtime/from_config.go

```diff
@@ -12,6 +12,7 @@ import (
 	syncbuilder "github.com/open-feature/flagd/core/pkg/sync/builder"
 	"github.com/open-feature/flagd/core/pkg/telemetry"
 	flageval "github.com/open-feature/flagd/flagd/pkg/service/flag-evaluation"
+	flag_sync "github.com/open-feature/flagd/flagd/pkg/service/flag-sync"
```

Comment:
nitpick, to use the recommended [naming scheme](https://go.dev/blog/package-names) : 
```suggestion
	flagsync "github.com/open-feature/flagd/flagd/pkg/service/flag-sync"
```

Reply:
agree, done with the commit https://github.com/open-feature/flagd/pull/1237/commits/d7c5798048864dcc504fffafbdebbf61fbdf1938